ew/static/cfg.py

```diff
@@ -3165,6 +3168,14 @@
 hue_id_gold = "gold"
 hue_id_negative = "negative"
 
+# list of items corresponding to their haunt power bonus
+item_hauntpower = {
+    "kidneystone":1.3, #Only works on specific players, and they have to lose their item to you.
+    "sord":0.1, #jpeg artifacts are extremely unspooky and actively negate power.
+    "nemes":1.05, #semi historic cosmetic
+    "negapoudrin":1.15, #basic channel item
```

Comment:
a negapoudrin costs -1,000,000. As long as it can be destroyed, Its literally not cost effective to use it over the next best thing unless we widen the gaps between items -but thatd need to buff hauntpower a decent amount for the high end items. I'm not really sure what to do about it. I think, because of its cost, i'll make it 1.3x too. IT cna be easily changed in the future too.
I imagine the dominant strategy is to use some cheap spooky cosmetic except when somebody has got huge slime and/or some of the other conditions in !haunt align to increase your haunt power -and you'd reserve your more powerful haunt items, like the kidney stone or negapoudrin, for that.